website/screens/overview/releases/ReleasesPage.tsx

```diff
@@ -0,0 +1,143 @@
+import {
+  DxcHeading,
+  DxcParagraph,
+  DxcFlex,
+  DxcSpinner,
+  DxcGrid,
+  DxcLink,
```

Comment:
Please remove the imports of `DxcSpinner` and `DxcGrid`, since they are not needed.

Reply:
Done!